src/watchmaker/static/shell/bootstrappers/watchmaker-bootstrap.ps1

```diff
@@ -57,6 +57,23 @@ function Install-Python-MSI {
   Start-Process "msiexec.exe" -ArgumentList ${Arguments} -Wait
 }
 
+function Install-Python-Packages {
```

Comment:
I moved the pip upgrade back into Get-Python because it is actually an installation of a new package but rather an upgrade of a package that already comes with Python's MSI installation.